app/views/learning/show.html.slim

```diff
@@ -33,19 +33,15 @@ h1.govuk-heading-l My modules
     .govuk-grid-column-full
       h2.govuk-heading-m Future modules in this course
 
-    - @user.course.upcoming_modules.each do |mod|
-      .govuk-grid-column-one-third-from-desktop.govuk-grid-column-one-half
-        .govuk-card__flex-container
-          .govuk-card__flex-item id=mod.name
-            .govuk-card.govuk-card__height
-              .govuk-card__content
-                h3.govuk-heading-s
-                  = "Coming soon - " if mod.draft?
-                  = "Module #{mod.id}: #{mod.title}"
-                p.govuk-body= mod.description
-              .govuk-card__actions
-                - unless mod.draft?
-                  = govuk_link_to 'View more information about this module', course_overview_path + "#module-#{mod.id}-#{mod.title.downcase.parameterize}"
+      .grid-container
+        - @user.course.upcoming_modules.each do |mod|
+          .govuk-card
+            h3.govuk-heading-s
+              = "Coming soon - " if mod.draft?
```

Comment:
I think this PR is the right one in which to attempt this: `TrainingModule#cards_title`

Reply:
Something similar is being used in `training_modules#index`

Reply:
I've added something. Let me know if that's what you were describing